### PR TITLE
Add "Ausaf Ahmed" to Logging and Metrics Reviewers

### PR DESCRIPTION
**Pull request by aqstack**
I am going to start actively contributing to logging and metrics, and am interested in becoming an approver down the road.

### CODE REVIEW

**Review by ctlong — APPROVED** [review]

**Comment by ctlong** [comment]
@acrmp @mkocher @ameowlia please take a look & approve when you get a chance 🙏

File: toc/working-groups/app-runtime-platform.md [hunk 1]
```diff
@@ -239,6 +239,8 @@ areas:
   reviewers:
   - name: Felix Hambrecht
     github: fhambrec
+  - name: Ausaf Ahmed
+    github: ausafqa
```
Comment (beyhan) [inline toc/working-groups/app-runtime-platform.md:243]: When I look for https://github.com/ausafqa I get 404. There is no such user. Is the github user correct?

Comment (aqstack) [reply toc/working-groups/app-runtime-platform.md]: I have updated the user handle.

Comment (aqstack) [reply toc/working-groups/app-runtime-platform.md]: @beyhan PTAL when you get a chance. Thanks!

**Review by beyhan — COMMENTED** [review]
@aqstack could you please add yourself also to the https://github.com/cloudfoundry/community/blob/main/org/contributors.yml. Sorry, I didn't notice this last time. Otherwise all looks fine now.

**Review by ameowlia — APPROVED** [review]

**Pull request merged**